CRAMv3.tex

```diff
@@ -440,7 +440,8 @@ \section{\textbf{Container structure}}
 \textbf{Data type} & \textbf{Name} & \textbf{Value}
 \tabularnewline
 \hline
-int32 & length & byte size of the container data (blocks)\tabularnewline
+int32 & length & the sum of the byte lengths of all blocks in this container; 
```

Comment:
"byte lengths of all blocks" may be ambiguous.  Are we talking the size of all data[] parts of blocks, or also including the block structure itself?  We know it's the latter, but it may not be clear from this wording given "byte lengths" and the fact that blocks have a "byte[]" array in their structure.

To be totally unambiguous, maybe use "the sum of the lengths of all blocks (both structures and data) in this container".